### ISSUE TITLE
Replace appended to With

### ISSUE THREAD

**Issue by worker109** [matheusfaustino/textfast#6]
On some text boxes the **With** text is appended to the **Replace** text after the replacement occurs, e.g.

**Replace**: e@
**With**: email@domain.com

**Expected result in text box:**
email@domain.com

**Actual result in text box:**
email@domain.come@

Can be re-created at [https://auspost.com.au/auth/login](https://auspost.com.au/auth/login) 

(Let me know if you need a hand, I'm happy to have an attempt at a fix)

**Comment by Rackmani** [comment]
yes this happens a fair bit sadly on many webpages  ...any chance this can be fixed ?
Otherwise this extn is great
